application/src/main/java/org/cardanofoundation/ledgersync/dto/GenesisData.java

```diff
@@ -20,7 +20,8 @@ public class GenesisData {
     List<Tx> txs;
     List<TxOut> txOuts;
     List<SlotLeader> slotLeaders;
-    CostModel costModel;
+    CostModel alonzoCostModel;
+    CostModel conwayCostModel;
```

Comment:
Do you need to keep costModel in two separate fields or just keep one as both values are merged to get the final cost model ?